src/sentry/static/sentry/app/views/projectIssueTracking.jsx

```diff
@@ -44,6 +44,9 @@ const IssuePluginConfiguration = React.createClass({
     return (
       <div>
         {this.props.plugins.map((plugin) => {
+          let pluginCls = (window.SentryPlugins[plugin.id] || DefaultPlugin);
```

Comment:
this part should be abstracted into a plugin loader/cache and should be responsible for:

- instantiating plugins
- caching assets (e.g. the .js file)